src/js/loginPage.js

```diff
@@ -0,0 +1,5 @@
+import '../scss/loginPage.scss';
+
+if (typeof (module.hot) !== 'undefined') {
+    module.hot.accept(); // eslint-disable-line no-undef
+  }
```

Comment:
Spacing. I believe that closing bracket should be the first character in this line

Reply:
You right :)